OSDOCS#11933: About gathering information about the workloads in your multi-architecture cluster (#83859)
openshift/openshift-docs: modules/multi-arch-gather-info-about-workloads.adoc

//Module included in the following assemblies
//

> //post_installation_configuration/multiarch-tuning-operator.adoc

Author: Updated as per the suggestion.

:_mod-docs-content-type: CONCEPT
[id="multi-architecture-gather-info-about-workloads_{context}"]

> = Gathering information about the workloads in your multi-architecture cluster

Reviewer: ```suggestion
= Multiarch Tuning Operator pod labels and architecture support overview
```

Author: Updated as per the suggestion.

> After installing the Multiarch Tuning Operator, you can verify the multi-architecture support for workloads in your cluster. The Multiarch Tuning Operator adds pod labels that allow you to identify and manage pods based on their architecture compatibility. These labels are automatically set on the newly created pods to provide insights into their architecture support.

Author: Updated as per the suggestion.

> After installing the Multiarch Tuning Operator, you can verify the multi-architecture support for workloads in your cluster. When you create a pod, the Multiarch Tuning Operator adds pod labels that allow you to quickly identify and manage pods based on their architecture compatibility. These labels are automatically set on patched pods to provide insights into their architecture support.

Reviewer 2: ```suggestion
After installing the Multiarch Tuning Operator, you can verify the multi-architecture support for workloads in your cluster. When you create a pod, the Multiarch Tuning Operator adds pod labels that allow you to quickly identify and manage pods based on their architecture compatibility. These labels are automatically set on newly created pods to provide insights into their architecture support.
```

Author: Updated as per the suggestion.

> After installing the Multiarch Tuning Operator, you can verify the multi-architecture support for workloads in your cluster. When you create a pod, the Multiarch Tuning Operator adds pod labels that allow you to identify and manage pods based on their architecture compatibility. These labels are automatically set on the newly created pods to provide insights into their architecture support.

Reviewer 2: Can we remove 'When you create a pod,'?

Author: Updated as per the suggestion. @aleskandro

> When a pod is created, the Multiarch Tuning Operator adds the labels described in the following table:

Reviewer: ```suggestion
The following table describes the labels that the Multiarch Tuning Operator adds when you create a pod:
```

Author: Updated as per the suggestion.

> .Pod labels added by the Multiarch Tuning Operator when a pod is created

Author: Updated as per the suggestion.

[%autowidth,options="header"]
|====

|Label |Description

|`multiarch.openshift.io/multi-arch: ""` |Indicates that a pod supports multiple architectures.
|`multiarch.openshift.io/single-arch: ""` |Indicates that a pod supports only a single architecture.
|`multiarch.openshift.io/arm64: ""` |Indicates that a pod supports the `arm64` architecture.
|`multiarch.openshift.io/amd64: ""` |Indicates that a pod supports the `amd64` architecture.
|`multiarch.openshift.io/ppc64le: ""` |Indicates that a pod supports the `ppc64le` architecture.
|`multiarch.openshift.io/s390x: ""` |Indicates that a pod supports the `s390x` architecture.

> |`multirach.openshift.io/node-affinity: set` |Indicates that the `node-affinity` requirement is already configured.

Reviewer 2: ```suggestion
|`multirach.openshift.io/node-affinity: set` |Indicates that the operator has set the node affinity requirement for the architecture
```

Author: Updated as per the suggestion.

> |`multirach.openshift.io/node-affinity: not-set` |Indicates that the `node-affinity` requirement is not configured.

Reviewer 2: ```suggestion
|`multirach.openshift.io/node-affinity: not-set` |Indicates that the operator did not set the node affinity requirements (for example, the pod was already having a node affinity for the architecture set)
```

Author: Updated as per the suggestion.

|`multiarch.openshift.io/scheduling-gate: gated` |Indicates that the pod is gated.
|`multiarch.openshift.io/scheduling-gate: removed` |Indicates that the pod gate has been removed.

> |`multiarch.openshift.io/inspection-error: ""` |Indicates that an error has occurred while inspecting the images.

Reviewer 2: ```suggestion
|`multiarch.openshift.io/inspection-error: ""` |Indicates that an error has occurred while building the node affinity requirements.
```

Author: Updated as per the suggestion.
